src/System.Threading.Tasks.Extensions/pkg/System.Threading.Tasks.Extensions.pkgproj

```diff
@@ -11,8 +11,7 @@
     </ProjectReference>
     <ProjectReference Include="..\src\System.Threading.Tasks.Extensions.csproj" />
 
-    <InboxOnTargetFramework Include="netcoreapp2.0" />
```

Comment:
You will also need to suppress the validation similar to https://github.com/stephentoub/corefx/blob/master/src/System.Reflection.Metadata/pkg/System.Reflection.Metadata.pkgproj#L18

Reply:
As I mentioned in the issue, this may not be needed due to the assembly versioning its minor.  We recognize a minor version bump as the intent to go out of box.

Reply:
> You will also need to suppress the validation

If it was a problem, would I have seen errors when doing build --allconfigurations?  That completed successfully.

Reply:
Yeah, you would have seen the error.